### PR TITLE
Removed BB dep from url service

### PR DESCRIPTION
**Pull request by Dave3of5**
My first PR for the project. Started off with a small change for #14882. It's only a single file (two actually) change as was mentioned in the Issue.

I also fixed a typo in a function name.

### CODE REVIEW

**Comment by ErisDS** [comment]
Hey @Dave3of5, thank you for this PR, really appreciate you taking the time to help out. I've just updated the [original issue](https://github.com/TryGhost/Ghost/issues/14882) to be a bit clearer that we need to remove catch predicates first (and why). I'll hold onto this PR and merge it once that work is done.

**Comment by Dave3of5** [comment]
@ErisDS Got it, I'll look at creating some PR's for removing the catch predicates then first.

**Review by ErisDS — APPROVED** [review]

File: ghost/core/core/server/services/url/Resources.js [hunk 1]
```diff
@@ -95,28 +94,28 @@ class Resources {
      * 3 event listeners connected to "_onResourceUpdated" handler and it's 'tag.edited', 'tag.attached', 'tag.detached' events
      * 1 event listener connected to  "_onResourceRemoved" handler and it's 'tag.deleted' event
      */
-    initEvenListeners() {
+    async initEventListeners() {
```
Comment (daniellockyer) [inline ghost/core/core/server/services/url/Resources.js:97]: I think we can do without the `async` here, and the corresponding `await`s in `ghost/core/core/server/services/url/UrlService.js` - this function doesn't do anything asynchronous, or return a Promise 🙂 (an upcoming lint rule addition to Ghost would have caught this)

Comment (Dave3of5) [reply ghost/core/core/server/services/url/Resources.js]: Good spot fix in 5371dc4

**Review by daniellockyer — APPROVED** [review]
@ErisDS this should be fine now 🙂

**Pull request merged**